### ISSUE TITLE
Export menu gets clipped after updating Highcharts

### ISSUE THREAD

**Issue by LaniMoo** [highcharts/highcharts#10361]
#### Expected behaviour
Export menu does not get clipped when chart heights are smaller than the export menu height.

#### Actual behaviour
After updating from 6.2.0 to 7.0.3, the export menus now get clipped. It's [always been like that](https://github.com/highcharts/highcharts/issues/9998) by default, but I was always able to work around it by tweaking the CSS. Now it looks like `overflow: hidden` is being applied through the code so I can no longer override it. 

I think this is the culprit: https://github.com/highcharts/highcharts/blob/a9d77d8d456320d46b9bc982f8de3572650e8a62/js/parts/Chart.js#L1175-L1189

#### Live demo with steps to reproduce
1. Open export menu.

6.2.0: http://jsfiddle.net/lanimelrok/xge30zaw/
7.0.3: http://jsfiddle.net/lanimelrok/sg0qyrjf/

#### Product version
7.0.3

**Comment by pawelfus** [comment]
Hi @LaniMoo 

Thanks for reporting. Now `overflow` is  set on the container too, you can overwrite it too: http://jsfiddle.net/BlackLabel/9q38rd1j/

```css
#container, svg {
  overflow: visible !important;
}
```

**Comment by sebastianbochan** [comment]
**Internal note**
FYI @TorsteinHonsi 

Bisect leads me to commit: https://github.com/highcharts/highcharts/commit/02b559d and is related with a few fixed by it bugs.

**Comment by LaniMoo** [comment]
Oh gosh, why didn't I think of that... thanks so much, @pawelfus!!

**Comment by LaniMoo** [comment]
> Hi @LaniMoo
> 
> Thanks for reporting. Now `overflow` is set on the container too, you can overwrite it too

Thank you, @pawelfus. Unfortunately that solution doesn't work for me because I'm using Angular, so the element that the chart renders to doesn't actually have an ID:

Highcharts in Angular
6.2.0: https://codesandbox.io/s/k2nnvywv3o
7.0.3: https://codesandbox.io/s/lpv6lx8jzz

I suppose I could go back and add an ID everywhere I use a chart, but that seems a bit cumbersome? 



> Maybe we should consider an option like exporting.outside similar to tooltip.outside ?

@sebastianbochan and @TorsteinHonsi I previously created an issue for this: https://github.com/highcharts/highcharts/issues/9998

**Comment by pawelfus** [comment]
Ah, Angular :) Alternatively you can use `chart.load` event to change that CSS: https://codesandbox.io/s/2on7yno3mr

Snippet:
```javascript
        events: {
          load: function () {
            Highcharts.css(this.container.parentNode, { overflow: 'visible' });
          }
        }
```

**Comment by LaniMoo** [comment]
Thank you, @pawelfus! Worked perfectly.

**Comment by stevethemacguy** [comment]
@pawelfus. I have the same issue with point markers no longer being able to overflow the chart area due to this change. Why not leave the CSS up to developers instead of adding inline styles to the parent container? Forcing developers to use an !important tag isn't an acceptable solution for production applications, especially considering this is a regression in HC 7.

**Comment by pawelfus** [comment]
I don't think we need a separate ticket - it's the same issue, regression mentioned by @sebastianbochan above: https://github.com/highcharts/highcharts/issues/10361#issuecomment-474784511 (which fixed many other issues, see: 02b559d).

@TorsteinHonsi - what do you think about limiting when `overflow: 'hidden'` is set (like checking parents) or just letting users set `overflow` for the container in chart config?

**Comment by TorsteinHonsi** [comment]
I think we can rather temporarily set `overflow` to `visible` when showing the menu, since the original issue was related to resizing. I'll check.
